### PR TITLE
[Product Partnerships] Added AWS Workspaces web

### PR DESCRIPTION
**Pull request by rahul188**
# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [X] Did you check you NRQL syntax? - Does it work?
- [X] Did you include a Data source and Documentation reference?
- [X] Are all documentation links publicly accessible?
- [X] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [X] Did you check your descriptive content for spelling and grammar errors?
- [X] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [X] Does the PR contain a screenshot for each of your dashboards?
- [X] Do your screenshots show data?
- [X] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [X] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/aws/amazon-workspaces-web/config.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+slug: amazon-workspaces-web
+description: |-
+  ## What is Amazon WorkSpaces Web?
+  Amazon WorkSpaces Web is an on-demand, fully managed, Linux-based service designed to facilitate secure browser access to internal websites and software-as-a-service (SaaS) applications. Access the service from existing web browsers, without the administrative burden of infrastructure management, specialized client software, or virtual private network (VPN) solutions.
+  ### Get started!
+  Start monitoring Amazon WorkSpaces Web by connecting Amazon Web Services (AWS) to New Relic! Take a look at our documentation for information on how to set up this integration.
+  ### More info
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-workspaces-web-monitoring-integration/) to learn more about New Relic monitoring for Amazon WorkSpaces web.
```
Comment (d3caf) [inline quickstarts/aws/amazon-workspaces-web/config.yml:8]:
```suggestion
  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-workspaces-web-monitoring-integration/) to learn more about New Relic monitoring for Amazon WorkSpaces Web.
```

File: dashboards/amazon-workspaces-web/amazon-workspaces-web.json [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+              "column": 1,
+              "row": 1,
+              "width": 3,
+              "height": 3
+            },
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "rawConfiguration": {
+              "text": "# Amazon Workspace Web\nAmazon WorkSpaces Web is an on-demand, fully managed, Linux-based service designed to facilitate secure browser access to internal websites and software-as-a-service (SaaS) applications. Access the service from existing web browsers, without the administrative burden of infrastructure management, specialized client software, or virtual private network (VPN) solutions."
```
Comment (d3caf) [inline dashboards/amazon-workspaces-web/amazon-workspaces-web.json:21]:
```suggestion
              "text": "# Amazon WorkSpaces Web\nAmazon WorkSpaces Web is an on-demand, fully managed, Linux-based service designed to facilitate secure browser access to internal websites and software-as-a-service (SaaS) applications. Access the service from existing web browsers, without the administrative burden of infrastructure management, specialized client software, or virtual private network (VPN) solutions."
```

File: dashboards/amazon-workspaces-web/amazon-workspaces-web.json [hunk 3]
```diff
@@ -0,0 +1,135 @@
+{
+    "name": "AWS Workspace Web",
```
Comment (d3caf) [inline dashboards/amazon-workspaces-web/amazon-workspaces-web.json:2]:
```suggestion
    "name": "Amazon WorkSpaces Web",
```

File: dashboards/amazon-workspaces-web/amazon-workspaces-web.json [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+    "description": null,
+    "pages": [
+      {
+        "name": "AWS Workspace Web",
```
Comment (d3caf) [inline dashboards/amazon-workspaces-web/amazon-workspaces-web.json:6]:
```suggestion
        "name": "Amazon WorkSpaces Web",
```

File: dashboards/amazon-workspaces-web/amazon-workspaces-web.png
Comment (d3caf) [inline dashboards/amazon-workspaces-web/amazon-workspaces-web.png:1]: We need to update this screenshot to display the correct capitalization for the product: Amazon WorkSpaces Web

**Review by d3caf — COMMENTED** [review]
Need to make some updates to the product naming in this one. It should be consistently spelled/capitalized like `Amazon WorkSpaces Web`

File: quickstarts/aws/amazon-workspaces-web/config.yml (hunk 1 above)
Comment (rahul188) [reply quickstarts/aws/amazon-workspaces-web/config.yml]: @d3caf, I made the suggested changes.

File: dashboards/amazon-workspaces-web/amazon-workspaces-web.json (hunk 2 above)
Comment (rahul188) [reply dashboards/amazon-workspaces-web/amazon-workspaces-web.json]: @d3caf, I made the suggested changes.

File: dashboards/amazon-workspaces-web/amazon-workspaces-web.json (hunk 3 above)
Comment (rahul188) [reply dashboards/amazon-workspaces-web/amazon-workspaces-web.json]: @d3caf, I made the suggested changes.

File: dashboards/amazon-workspaces-web/amazon-workspaces-web.json (hunk 4 above)
Comment (rahul188) [reply dashboards/amazon-workspaces-web/amazon-workspaces-web.json]: @d3caf, I made the suggested changes.

File: dashboards/amazon-workspaces-web/amazon-workspaces-web.png
Comment (rahul188) [reply dashboards/amazon-workspaces-web/amazon-workspaces-web.png:1]: @d3caf, I made the suggested changes.

Comment (rahul188) [reply dashboards/amazon-workspaces-web/amazon-workspaces-web.png:1]: @d3caf, I made the suggested changes.

**Review by d3caf — APPROVED** [review]

**Pull request merged**